aws/client/default_retryer.go

```diff
@@ -37,14 +37,29 @@ func (d DefaultRetryer) RetryRules(r *request.Request) time.Duration {
 		retryCount = 13
 	}
 
-	delay := (1 << uint(retryCount)) * (rand.Intn(30) + 30)
+	minTime := 30
+	if d.shouldThrottle(r) {
+		minTime = 1000
+	}
+
+	delay := (1 << uint(retryCount)) * (rand.Intn(minTime) + 30)
```

Comment:
`delay := (1 << uint(retryCount)) * (rand.Intn(minTime) + 30)`
should be 
`delay := (1 << uint(retryCount)) * (rand.Intn(30) + minTime)`?